common/templates/error.njk

```diff
@@ -4,6 +4,8 @@
 
 {% block organisationSwitcher %}{% endblock %}
 
+{% block beforeContent %}{% endblock %}
```

Comment:
What is this empty block hiding from the error page?

Reply:
This prevents the breadcrumb being shown on the default error page, I popped a comment in the commit description about this change :)